src/test/java/org/inferred/freebuilder/processor/MapPropertyFactoryTest.java

```diff
@@ -214,15 +214,16 @@ public void testPut_primitiveKey_nullValue() {
 
   @Test
   public void testPut_duplicate() {
-    thrown.expect(IllegalArgumentException.class);
-    thrown.expectMessage("Key already present in items: bar");
     behaviorTester
         .with(new Processor())
         .with(MAP_PROPERTY_TYPE)
         .with(new TestBuilder()
-            .addLine("new com.example.DataType.Builder()")
+            .addLine("com.example.DataType value = new com.example.DataType.Builder()")
             .addLine("    .putItems(\"bar\", \"baz\")")
-            .addLine("    .putItems(\"bar\", \"bam\");")
+            .addLine("    .putItems(\"bar\", \"bam\")")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String literal "    .putItems(\"bar\", \"bam\")" appears 4 times in this file; the first occurrence is on line 223](https://www.codacy.com/app/Chris-Purcell-39/FreeBuilder/file/2409332805/issues/source?bid=3146243&fileBranchId=3234738#l223)